src/scripts/cf/form-tags/Tag.ts

```diff
@@ -210,17 +272,34 @@ namespace cf {
 			let isValid: boolean = true;
 			let valueText: string = value.text;
 
-			if(this.pattern){
-				isValid = this.pattern.test(valueText);
```

Comment:
updated my pull request to accept either HTML5 pattern attr or cf-validation-matches attr